src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -13450,17 +13450,6 @@ void Assembler::movsbq(Register dst, Register src) {
   emit_int24(0x0F, (unsigned char)0xBE, (0xC0 | encode));
 }
 
-void Assembler::movslq(Register dst, int32_t imm32) {
```

Comment:
You can remove the corresponding declaration in the header file.

Reply:
Right, removed.